os/arch/arm/src/amebad/amebad_spi.c

```diff
@@ -98,6 +97,10 @@ struct amebad_spidev_s {
 	uint32_t actual;            /* Actual clock frequency */
 
 	spi_t spi_object;
+#ifdef CONFIG_SPI_CS
+	gpio_t gpio_cs0;
+	gpio_t gpio_cs1;
+#endif
```

Comment:
In the else case could you keep one gpio_t instance for the single slave? It can be used to select and deselect it.